doc/schemas/app_openapi.json

```diff
@@ -990,7 +990,7 @@
         "responses": {
           "200": {
             "content": {
-              "application/json": {
+              "text/plain": {
```

Comment:
I tried this yesterday and Sphinx would not generate example responses anymore. If we want to have those, it may be worth another look to make sure it works.

Reply:
I think that's fine. The auto generated example responses are nice-to-have, but they're not particularly helpful. At some point we'll add explicit examples.